src/main/java/org/fundacionjala/coding/yerel/FizzBuzz.java

```diff
@@ -0,0 +1,42 @@
+package org.fundacionjala.coding.yerel;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * this is a class for find number Fizz or Fuzz.
+ */
+public class FizzBuzz {
+    private static final int NUMBER_THREE = 3;
+    private static final int NUMBER_FIVE = 5;
+    private static final int NUMBER_ZERO = 0;
+
+    /**
+     * @param number to find String FizzBuzz.
+     * @return Fizz o Fuzz values.
+     */
+    public String multiplesForNumber(final int number) {
+        if (number % NUMBER_FIVE == NUMBER_ZERO && number % NUMBER_THREE == NUMBER_ZERO) {
+            return "FizzBuzz";
+        }
+        if (number % NUMBER_FIVE == NUMBER_ZERO || String.valueOf(number).contains("5")) {
```

Comment:
You can create a local variable for String.valueOf(number)
and replace the occurrencies.